tools/rabbitmq-quorum-migration.sh

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/bash
+
```

Comment:
`set -euo pipefail`

Reply:
I've used -ex until now, testing pipefail and will see how it turns out on Monday morning

Reply:
Seems fine

Reply:
No pipefail?

Reply:
Reverted my changes :facepalm:
I'll quickly retest just to be sure

Reply:
Good thing I checked. Setting pipefail permenantly kills the AIO somehow. It'll shut it down and then fail to start up again

Reply:
Going to stick with -ex, I think that's the best we can do.